examples/linear_growth/linear_growth_params.ini

```diff
@@ -0,0 +1,82 @@
+output_root = linear_growth
```

Comment:
You don't need this file - it's just a copy of the input, which you also have below.